lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,69 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+
+      DEFAULT_STEP = 1 
+      APP_MIGRATION_DIRECOTRY = "apps/web/db/migrations".freeze
+      MIGRATION_DIRECTORY = "db/migrations".freeze
```

Comment:
I could call Lotus::Model::Migration:: MIGRATION_DIRECTORY in this case I think Lotus::Model::Migration should have Lotus::Model::Migration::APP_MIGRATION_DIRECTORY as well.